src/frontend/staff-portal/src/app/components/jj-dispute-assignments/jj-dispute-assignments.component.ts

```diff
@@ -70,7 +53,7 @@ export class JJDisputeAssignmentsComponent implements OnInit, AfterViewInit {
    }
 
   ngOnInit(): void {
-    this.jjList = this.jjList.sort((a: JJTeamMember, b: JJTeamMember) => { if (a.name < b.name) { return -1; } else { return 1 } });
+    this.jjDisputeService.jjList = this.jjDisputeService.jjList.sort((a: JJTeamMember, b: JJTeamMember) => { if (a.name < b.name) { return -1; } else { return 1 } });
```

Comment:
why not component based list ?

Reply:
it is shared with more than one component

Reply:
you can sort the jjlist inside the jjDisputeService

Reply:
anyway it is not a must for changing this